cypress/e2e/gui-tests/gui-home.cy.js

```diff
@@ -0,0 +1,21 @@
+describe('Home Page', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  context('Initial Load', () => {
+    it('should display a "Loading..." fallback initially', () => {
+      cy.contains('Loading...').should('be.visible');
```

Comment:
Idealmente, você deveria forçar um atraso na requisição para testar tal estado transitório.

Isso torna tais testes determinísticos, visto que damos tempo para o Cypress verificar tal elemento transitório antes dele desaparecer.

Recomendo a leitura do seguinte conteúdo: [Como simular a demora em uma requisição com Cypress](https://talkingabouttesting.com/2021/02/26/como-simular-a-demora-em-uma-requisicao-com-cypress/)